app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: "str", health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, Hidden: {self.hidden}}}")
+
+    def __repr__(self) -> str:
+        return self.__str__()
+
+    @staticmethod
+    def die(animal: "Animal") -> None:
```

Comment:
This method 'die' should be an instance method, not a static method. It operates on the instance of the class, not the class itself. You should use `self` instead of passing `animal` as an argument. This is related to the 'Use the class name to get access to the class attributes' point in the checklist.